fe/src/components/DatePicker.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div>
+    <div class="date-wrapper">
+      <VueDatePicker
+        class="date-picker"
+        v-model="startDate"
+        :min-date="getNextDay(currentDate)"
+        :max-date="endDate || null"
+        placeholder="Start Date"
+        @input="setStartDateTo2PM"
+      />
+      <VueDatePicker
+        class="date-picker"
+        v-model="endDate"
+        :min-date="startDate ? getNextDay(startDate) : currentDate"
```

Comment:
Current date is not allowed
![image](https://github.com/framgia/pj_sim-sunbnb/assets/137018413/94588d0c-04f5-419a-ad5d-342a67758837)

Reply:
Will do.